_data/projects/maven-test-bench.yml

```
@@ -8,7 +8,7 @@ tags:
 - performance
 - quickperf
 upforgrabs:
-  name: ":thumbsup: good first issue"
+  name: : "thumbsup: good first issue"
```

@jeanbisutti Please remove this extra colon outside ""
```suggestion
  name: ":thumbsup: good first issue"
```